tests/xtro-sharpie/api-annotations-dotnet/macOS-AppKit.ignore

```diff
@@ -1687,3 +1687,7 @@
 !missing-type! NSPDFPanel not bound
 !missing-type! NSPICTImageRep not bound
 !missing-type! NSPersistentDocument not bound
+
+# the method is a static constructor that return a label for a New item, they used New not to represent the intent of the
+# method, but what label it returns (probably a new objc developer at apple)
+!missing-release-attribute-on-return-value! AppKit.NSMenuItemBadge AppKit.NSMenuItemBadge::CreateNewItems(System.IntPtr)'s selector's ('newItemsWithCount:') Objective-C method family ('new') indicates that the native method returns a retained object, and as such a '[return: Release]' attribute is required.
```

Comment:
Did you test that the selector in question doesn't return a retained object? AFAIK that's automatic in the Objective-C compiler, so unless Apple's developer did something specific to get a different behavior, we'll get a retained object.

Reply:
Yes, this is from someone that did not know about Objc, the Badge it creates is fo  "New Items" as opossed to Update Items.. a very poorly chosen API if you consider the objc common practice :/